platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -21,10 +21,72 @@ Switch your build to use Gradle @version@ by updating the [Wrapper](userguide/gr
 
 See the [Gradle 8.x upgrade guide](userguide/upgrading_version_8.html#changes_@baseVersion@) to learn about deprecations, breaking changes, and other considerations when upgrading to Gradle @version@.
 
-For Java, Groovy, Kotlin, and Android compatibility, see the [full compatibility notes](userguide/compatibility.html).   
+For Java, Groovy, Kotlin, and Android compatibility, see the [full compatibility notes](userguide/compatibility.html).
 
 ## New features and usability improvements
 
+### Error and warning reporting improvements
+
+#### Ambiguous Artifact Transformation chains are detected and reported
+
+Previously, when two or more equal-length chains of <<artifact_transforms.adoc#sec,artifact transforms>> produced compatible variants to satisfy a resolution request, Gradle would arbitrarily and silently select one.
+Gradle now emits a deprecation warning for this case.
```

Comment:
I find it odd that we are talking about deprecations here. We usually only talk about them in the upgrade guide.

To me, there are two changes in this PR:
1. We deprecated some ambiguity in when selecting artifact transforms
2. We improved the message you get when you have ambiguous artifact transforms that were already previously detected.

I think 1. belongs in the upgrade guide and 2. belongs in the release notes. I would not talk about 1. in the release notes